desktop/core/src/desktop/log/__init__.py

```diff
@@ -201,7 +201,7 @@ def basic_logging(proc_name, log_dir=None):
         h.setLevel(lvl)
 
     from desktop.conf import DATABASE_LOGGING
```

Comment:
this line here to avoid circular dependency?

Reply:
If yes, then I think good to add a comment for it.